tests/modules/module-with-transform-schema.js

```diff
@@ -15,10 +15,21 @@ module.exports = async function plugin(fastify) {
   })
 }
 
-module.exports.transformSchemaForSwagger = ({ schema, url }) => {
+module.exports.transformSchemaForSwagger = ({ schema, url } = {}) => {
+  if (!schema) {
+    return {
+      url,
+      schema: {
+        hide: true,
+      },
+    }
+  }
   if (schema.hide) {
     // if a route is set as hidden its schema should not be modified
-    return schema
+    return {
+      schema,
+      url,
```

Comment:
why adding the url now? 🤔

Reply:
With fastify swagger at v4, the transform function is changed and it requires to return an object with schema and url. It's a bug on a test of the fastify-v4 branch that I discovered with this changes